app/lib/actions/katello/repository/update.rb

```diff
@@ -9,10 +9,10 @@ def plan(repository, repo_params)
           if (SETTINGS[:katello][:use_cp] && SETTINGS[:katello][:use_pulp]) && repository.library_instance?
             plan_action(::Actions::Candlepin::Product::ContentUpdate,
                         :content_id => repository.content_id,
-                        :name => repository.name,
+                        :name => repository.content.name,
```

Comment:
Oh right there isn't.  I was thinking you were updating the content url, but you shouldn't be doing that and that would be bad.  I don't think there is a case to do that (among the attributes listed here).